processes-webapp/src/main/webapp/vue-app/processes/components/Processes.vue

```diff
@@ -111,6 +120,30 @@ export default {
       this.alert = true;
       window.setTimeout(() => this.alert = false, 5000);
     },
+
+    
+    addNewWorkFlow(workflow) {
+      this.saving = true;
```

Comment:
```suggestion
      this.saving = true;
```
variable is not declared